packages/components/nodes/Etherscan/Etherscan.ts

```diff
@@ -0,0 +1,217 @@
+                    },
+                    {
+                        label: 'Get Contract ABI for Verified Contract Source Codes',
+                        name: 'getContractABI',
+                        description: 'Returns the Contract Application Binary Interface ( ABI ) of a verified smart contract.'
+                    },
+                ],
+                
+                default: 'getEtherBalance'
+            },
+        ] as INodeParams[];
+        this.networks = [
+            {
+                label: 'Network',
+                name: 'network',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Ethereum Mainnet',
+                        name: 'homestead',
+                    },
+                ],
+                default: 'homestead',
+            },
+        ] as INodeParams[];
+        this.credentials = [
+            {
+                label: 'Credential Method',
+                name: 'credentialMethod',
+                type: 'options',
+                options: [
+                    {
+                        label: 'Etherscan API Key',
+                        name: 'etherscanApi',
+                    },
+                ],
+                default: 'etherscanApi',
+            },
+        ] as INodeParams[];
+        this.inputParameters = [
+            {
+                label: 'Address',
+                name: 'address',
+                type: 'string',
+                description: 'The address to check for balance'
+            },
+        ] as INodeParams[];
+    }
+    async run(nodeData: INodeData): Promise<INodeExecutionData[] | null> {
+        const actionData = nodeData.actions;
+        const networksData = nodeData.networks;
+        const inputParametersData = nodeData.inputParameters;
+        const credentials = nodeData.credentials;
+        if (actionData === undefined || inputParametersData === undefined || credentials === undefined || networksData === undefined) {
+            throw new Error('Required data missing');
+        }
+        // GET api
+        const api = actionData.api as string;
+        
+        // GET network
+        const network = networksData.network as string;
+        // GET credentials
+        const apiKey = credentials.apiKey as string;
+    
+        // GET address
+        const address = inputParametersData.address as string;
+        const returnData: ICommonObject[] = [];
+        let responseData: any;
+        if (api === 'getEtherBalance') {
+            try {
+                const queryParameters = {
+                    module: 'account',
+                    action: 'balance',
+                    address,
+                    tag: 'latest',
+                    apikey: apiKey,
+                }
+                let url = '';
+                // Change url depending on network. See https://docs.etherscan.io/getting-started/endpoint-urls
+                if (network === 'homestead') {
```

Comment:
So intead of typing and checking string, we check for `network === NETWORK.MAINNET` in enums to be safer